README.md

```diff
@@ -12,8 +12,14 @@ This is the landing page for Simian CI.
 
 ## SASS
 
+_Make sure you run `jekyll serve` at least once before running any gulp task, as they depend on the that are on the _site folder_
+
 Jekyll compiles SASS when serving/building. Put your partials in the `_sass` folder, and your `main.scss` file in `css` folder. Your `main.scss` file should have the front matter declaration (dashes at the top of the file) to work correctly.
 
+We are using PurifyCSS to remove unused style rules, we made a gulp task for it, to start te watcher run:
```

Comment:
This is cool but I want some place that says like: "before start to develop run BLA and BLE to see the changes" and if we need to do something extra to see changes made on JS please clarify too